Engine/Source/ModuleDebugDraw.cpp

```diff
@@ -639,7 +639,7 @@ void ModuleDebugDraw::Draw(const float4x4& viewproj,  unsigned width, unsigned h
     if (mDrawGrid) {
        DrawGrid();
     }
-       DrawAxis();
+    //DrawAxis();
```

Comment:
If this is not needed anymore, I'd propose to remove it, or comment why it's been commented to have it clear. Maybe a TODO can help here.

Reply:
Ok! Now it doesn't make any sense to keep that, so I would delete it.
Also, would you remove the axis on the center of the grid or do we keep it to have a reference of the 0,0,0?

Reply:
Good question... I think it can be useful, but probably can be misleading/confusing for new users. I think best thing is  to remove it, but let's see what opinion have the leads @M1R4B3L @carlosgspsp